src/elements/common/sub-header/__tests__/SortButton.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { SortButtonBase } from '../SortButton';
+import Button from '../../../../components/button';
+import IconSort from '../../../../icons/general/IconSort';
+
+const intlMock = {
+    formatMessage: jest.fn().mockReturnValue('Sort'),
+};
+
+describe('Elements/SubHeader/SortButton', () => {
+    const getWrapper = () => shallow(<SortButtonBase intl={intlMock} />);
+
+    test('should render IconSort', () => {
+        const wrapper = getWrapper();
+        expect(wrapper.exists(IconSort)).toBe(true);
+    });
+
+    test('should have aria-label "sort"', () => {
```

Comment:
if you're putting the text in quotation marks, then I would capitalize to `Sort` so it's the exact text